File: TestFiles/test6.py
def main():
   run_game(82, 2)
   print("Thank you for playing! ")
   
def run_game(x,y):
   num = False 
   num = "String"
   num = [3]
   num = x
   done = False
   num_guess = 1
   guess = y
   while not done:
      guess += 1
      if (num == guess):
         done = True
         print("Correct! Number of attempts:")
         print(num_guess)
      else:
         num_guess = num_guess + 1
         if (guess < num):
            print("Too low")
         else:
            print("Too high")
   test()
   return

def test():
   a = 3
   a = True 
   a = 'c'
   return

main()

File: TestFiles/test6.java
package TestFiles;
public class test6 {
	public static void main(String[] args) {
		run_game(82,2);
		System.out.println("Thank you for playing! ");
	}

	public static void run_game(int x, int y) {
		boolean num = false;
		String C = "String";
		int[] D = {3};
		int E = x;
		boolean done = false;
		int num_guess = 1;
		int guess = y;
		while (!done) {
			guess += 1;
			if (E == guess) {
				done = true;
				System.out.println("Correct! Number of attempts:");
				System.out.println(num_guess);
			} else {
				num_guess = num_guess + 1;
				if (guess < E) {
					System.out.println("Too low");
				} else {
					System.out.println("Too high");
				}
			}
		}
		test();
		return;
	}

	public static void test() {
		int a = 3;
		boolean A = true;
		String B = "c";
		return;
	}

}